internal/bgp/bgpfrr/frr_test.go

```diff
@@ -0,0 +1,333 @@
+package bgpfrr
+
+import (
+	"flag"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"go.universe.tf/metallb/internal/bgp"
+)
+
+const TESTDATA = "testdata/"
```

Comment:
nit: const doesn't need to be uppercase

Reply:
doesn't need to be lower case either :) I can change it if you want. Is there a convention?

Reply:
That's good enough for me.